test/test_sync.cpp

```diff
@@ -261,13 +261,13 @@ TEST(SyncTimePoint, OperatorMinusEqDuration)
  */
 /*****************************************************************************/
 
-template <bool USE_MONOTONIC_CLOCK = false>
+template <bool TEST_MONOTONIC_CLOCK = false>
```

Comment:
I changed the name because the `USE_MONOTONIC_CLOCK` symbol sounded too confusing with a macrodefinition that the build files could provide.